src/platform/plugins/shared/unified_search/public/query_string_input/query_bar_top_row.tsx

```diff
@@ -565,6 +589,25 @@ export const QueryBarTopRow = React.memo(
     function renderCancelButton() {
       const buttonLabelCancel = strings.getCancelQueryLabel();
 
+      if (isBackgroundSearchEnabled) {
+        return (
+          <SplitButton
+            onClick={onClickCancelButton}
+            onSecondaryButtonClick={onClickSendToBackground}
+            aria-label={buttonLabelCancel}
+            size={shouldShowDatePickerAsBadge() ? 's' : 'm'}
+            iconType="cross"
+            data-test-subj="queryCancelSplitButton"
+            color="text"
+            secondaryButtonIcon="clock"
+            secondaryButtonAriaLabel={strings.getSendToBackgroundLabel()}
+            isSecondaryButtonDisabled={!canSendToBackground}
+          >
+            {buttonLabelCancel}
+          </SplitButton>
```

Comment:
One ask for the "Send to background" button, don't think we need a tooltip, but let's add a html label, so the text is displayed on mouse hover. This should be sufficient info to figure out what's the button doing ... in case somebody wonders, oh, nice a clock, what's the time please?